### PR TITLE
Drop checked_out capability in com_privacy

### PR DESCRIPTION
**Pull request by mbabker**
Pull Request for Issue #22722

### Summary of Changes

Privacy requests ended up not having a notion of editing that works with the checkout system.  So, drop the checkout related data.

### Testing Instructions

Apply patch, ensure columns removed, ensure things work.

### CODE REVIEW

File: installation/sql/mysql/joomla.sql [hunk 1]
```diff
@@ -1690,12 +1690,16 @@ CREATE TABLE IF NOT EXISTS `#__privacy_requests` (
   `request_type` varchar(25) NOT NULL DEFAULT '',
   `confirm_token` varchar(100) NOT NULL DEFAULT '',
   `confirm_token_created_at` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
-  `checked_out` int(11) NOT NULL DEFAULT '0',
-  `checked_out_time` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
   PRIMARY KEY (`id`),
   KEY `idx_checkout` (`checked_out`)
```
Comment (alikon) [inline installation/sql/mysql/joomla.sql]: remove index if you removed the field from the table
```suggestion

```

**Review by alikon — CHANGES_REQUESTED** [review]

File: installation/sql/mysql/joomla.sql (hunk 1 above)
Comment (mbabker) [reply installation/sql/mysql/joomla.sql]: Indexes removed

**Comment by alikon** [comment]
Tested successfully even if not reported from issues

**Comment by alikon** [comment]
Tested successfully even if not reported from issues

**Comment by alikon** [comment]
tested successfully

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 7226a2bce8bd08947453be145126e759580147ec<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22748">issues.joomla.org/tracker/joomla-cms/22748</a>.</sub>

**Comment by infograf768** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22748">issues.joomla.org/tracker/joomla-cms/22748</a>.</sub>

**Pull request merged**